src/lucky/renderable.cr

```diff
@@ -37,6 +37,10 @@ module Lucky::Renderable
     response.print
   end
 
+  private def handle_response(_response)
```

Comment:
I just learned this nice trick so that we can get the type that someone used. This will make it easier for people to debug:

```crystal
private def handle_response(_response : T) forall T
  {% raise <<-ERROR
  You must return a Lucky::Response from #{@type}. Instead, you returned #{T}.

  Try this...
    ▸ Make sure you use `render`, `redirect`, `json` at the end of your action.
  ERROR %}
end
```

I also modified the error a bit to match some of the newer error messages in the other Lucky libs. What do you think?

Reply:
Actually, you're right that it might be lost. How about we switch it to something like this:

`Your action returned #{T}, but it must return a Lucky::Response.`

I think it's a bit shorter, and this way the type they returned won't be buried. What do you think?

Reply:
Here's a screenshot of the message in action. I think we should keep `@type` in the message to provide context.  I'm not sure if `T` is helpful or confusing, as it will include Lucky::Response in output along with any other types that were incorrectly used.

<img width="1609" alt="screen shot 2017-12-11 at 10 57 06 pm" src="https://user-images.githubusercontent.com/9160106/33868145-d9dafc38-dec6-11e7-9016-0db40303dedb.png">

Reply:
I think that looks good. I think there are just a couple more things:

* Add the word `or` before `json`
* Indent arrow 2 spaces
* Consider adding another tip: "If you are using a conditional, make sure all branches return a Lucky::Response". Maybe that's obvious though. I'm not sure. What do you think?

Reply:
It's probably obvious to more experienced developers, but the extra nudge about conditionals totally would have been helpful to me when I was first learning Rails.

Reply:
I can definitely see a case where the extra heads up about conditionals would unstick someone.  I added it in the commit.